packages/plugin-kit/src/fs/deleteAllCustomTypeFiles.ts

```diff
@@ -0,0 +1,24 @@
+import {
+	buildCustomTypeDirectoryPath,
+	BuildCustomTypeDirectoryPathArgs,
+} from "./buildCustomTypeDirectoryPath";
+import { deleteProjectFile } from "./deleteProjectFile";
+
+export type DeleteAllCustomTypeFilesArgs = BuildCustomTypeDirectoryPathArgs;
+
+export async function deleteAllCustomTypeFiles(
```

Comment:
Since it also deletes the directory, maybe `deleteCustomTypeDirectory` is clearer? Naming confused me at first.

Reply:
Works for me!